src/index.js

```diff
@@ -65,6 +65,16 @@ function stringify(json) {
   return JSON.stringify(removeUndefinedObjects(typeof json.RAW_BODY !== 'undefined' ? json.RAW_BODY : json));
 }
 
+function stringifyParameter(param) {
+  if (param === null || isPrimitive(param)) {
+    return param;
+  } else if (Array.isArray(param) && param.every(isPrimitive)) {
+    return String(param);
+  }
```

Comment:
If its fully an array of primitives like `[1,2,3], running `String()` on it will turn it into `1,2,3`, otherwise we'll convert it into a string with JSON stringification.